app/src/main/res/layout/repeat_inputs.xml

```diff
@@ -39,6 +40,7 @@
 			android:textSize="@dimen/text_21"
 			android:textColor="@color/offWhite"
 			android:background="@null"
+		    	android:fontFamily="font-normal"
```

Comment:
Is the StaxLabeledInput style not setting these correctly?

Reply:
It wasn't on my device